### PR TITLE
ac 2: Corrected sem number for students of all batches

### PR DESCRIPTION
**Pull request by Anunaya07**
## Proposed changes:
Corrected sem number for students of all batches

## Types of changes

_Put an `x` in the boxes that apply_

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Aakash2611Wagh** [comment]
Approved it @akshatnema

File: FusionIIIT/templates/globals/usercard.html [hunk 1]
```diff
@@ -52,7 +53,8 @@
                         {{ request.user.extrainfo.student.programme }}
                         {{ request.user.extrainfo.student.batch }}
                         <br/>
-                        Sem - {{ request.user.extrainfo.student.curr_semester_no }}
+                        {% comment %} Sem - {{ user.extrainfo.student.curr_semester_no }} {% endcomment %}
```
Comment (akshatnema) [inline FusionIIIT/templates/globals/usercard.html]:
```suggestion
```

File: FusionIIIT/templates/globals/usercard.html [hunk 2]
```diff
@@ -32,7 +32,8 @@
                         {{ user.extrainfo.student.programme }}
                         {{ user.extrainfo.student.batch }}
                         <br/>
-                        Sem - {{ user.extrainfo.student.curr_semester_no }}
+                        {% comment %} Sem - {{ user.extrainfo.student.curr_semester_no }} {% endcomment %}
```
Comment (akshatnema) [inline FusionIIIT/templates/globals/usercard.html]:
```suggestion
```

File: FusionIIIT/applications/academic_procedures/views.py [hunk 3]
```diff
@@ -298,7 +298,8 @@ def academic_procedures_student(request):
         curr_sem_id = Semester.objects.get(curriculum = curr_id, semester_no = obj.curr_semester_no)
 
         try:
-            user_sem = obj.curr_semester_no
+            # user_sem = obj.curr_semester_no
+            user_sem = get_user_semester(request.user, ug_flag, masters_flag, phd_flag)
             # user_sem = get_user_semester(request.user, ug_flag, masters_flag, phd_flag)
```
Comment (akshatnema) [inline FusionIIIT/applications/academic_procedures/views.py]:
```suggestion
            user_sem = get_user_semester(request.user, ug_flag, masters_flag, phd_flag)
```

**Review by akshatnema — CHANGES_REQUESTED** [review]

**Review by akshatnema — APPROVED** [review]

**Pull request merged**